test-cases/longevity/longevity-ycsb-a-10M.yaml

```diff
@@ -0,0 +1,56 @@
+# avg load speed per 2*2*128 threads 44K ops/s each
+# 10M / 44K = ~3 minutes
+#
+# 240 = 4 hours * 60
+# 480 = 4 hours * 60
+# 10900 = 7 days * 24 hours a day * 60 minutes an hour
+test_duration: 480
+
+pre_create_keyspace: [
+  "CREATE KEYSPACE ycsb WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': 3 };",
+  "CREATE TABLE ycsb.usertable (y_id varchar primary key, field0 varchar, field1 varchar, field2 varchar, field3 varchar, field4 varchar, field5 varchar, field6 varchar, field7 varchar, field8 varchar, field9 varchar);"
```

Comment:
BTW, I have code doing this somewhere in SCT.

But I like this approach even better.